### PR TITLE
Feature/eliminate hos variants

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

File: xml/installation-installation-gui_installer.xml [hunk 1]
```diff
@@ -7,6 +7,8 @@
  xmlns:xi="http://www.w3.org/2001/XInclude"
  xmlns:xlink="http://www.w3.org/1999/xlink">
  <title>Installing via the GUI</title>
+ <!--FIXME: GUI installation will be completely redone. Waiting on new
+ Input Model descriptions and instructions to do any edits here. -->
```
Comment (sknorr) [inline xml/installation-installation-gui_installer.xml]: Does it make sense to keep this comment?

File: xml/installation-installation-ironic_oneview_integration.xml [hunk 2]
```diff
@@ -196,19 +195,14 @@ $ cd ~/scratch/ansible/next/ardana/ansible/
...
-     Ironic deploy images ("ir-deploy-*") were created automatically by HOS
+     Ironic deploy images ("ir-deploy-*") are created automatically by the
      installer. We will be using
      <emphasis role="bold">agent_pxe_oneview</emphasis> Ironic driver which
      needs <emphasis role="bold">ir-deploy-kernel</emphasis> and
-     <emphasis role="bold">ir-deploy-ramdisk</emphasis> images. Managed node
-     will be deployed using <emphasis role="bold">Ubuntu Trusty 14.04
-     BIOS</emphasis> image.
+     <emphasis role="bold">ir-deploy-ramdisk</emphasis> images.
+     The image used to deploy managed nodes will also be shown.
```
Comment (sknorr) [inline xml/installation-installation-ironic_oneview_integration.xml]: Mhm ... As it stands, this sounds like a non-information. Which image will it be?

File: xml/installation-installation-gui_installer.xml (hunk 1 above)
Comment (csymons-suse) [reply xml/installation-installation-gui_installer.xml]: No. It needs to go. I'm working on that section right now. Should I do something about it now? Can you get rid of it?

File: xml/installation-installation-ironic_oneview_integration.xml (hunk 2 above)
Comment (csymons-suse) [reply xml/installation-installation-ironic_oneview_integration.xml]: Should be

Ironic deploy images (boot image, ir-deploy-kernel, ir-deploy-ramdisk, ir-deploy-iso) are created automatically. The <emphasis role="bold">agent_pxe_oneview</emphasis> Ironic driver requires <emphasis role="bold">ir-deploy-kernel</emphasis> and <emphasis role="bold">ir-deploy-ramdisk</emphasis> images.

File: xml/installation-installation-gui_installer.xml (hunk 1 above)
Comment (csymons-suse) [reply xml/installation-installation-gui_installer.xml]: never mind. I'm a little apprehensive. But it seems like I should just be able to make these changes and push them to this branch. So I'm gonna do that.
UPDATE: I started to do that and got several conflicts. Doesn't seem like such a hot idea. How should this be handled?

File: xml/installation-installation-ironic_oneview_integration.xml (hunk 2 above)
Comment (csymons-suse) [reply xml/installation-installation-ironic_oneview_integration.xml]: fixed

File: xml/installation-installation-gui_installer.xml (hunk 1 above)
Comment (csymons-suse) [reply xml/installation-installation-gui_installer.xml]: fixed. Bite the bullet. Live a little. I think everything worked okay. I'll wait for the judgement.

**Comment by sknorr** [comment]
Thanks for fixing the stuff I remarked about!

**Pull request merged**